src/utils/get-options/base.js

```diff
@@ -0,0 +1,50 @@
+import Promise from 'pinkie';
+import convertToBestFitType from '../convert-to-best-fit-type';
+
+
+const DEFAULT_OPTIONS_SEPARATOR   = ',';
+const DEFAULT_KEY_VALUE_SEPARATOR = '=';
+
+const DEFAULT_ON_OPTION_PARSED = (key, value) => value;
+
+function parseOptionsString (optionsStr, optionsSeparator, keyValueSeparator) {
```

Comment:
Probably we can replace it by some npm module? Besides it can be simplify:
```js
function parseOptionsString (optionsStr, optionsSeparator, keyValueSeparator) {
    const splittedOptions = optionsStr.split(optionsSeparator);

     if (!splittedOptions.length)
        return null;

     return splittedOptions.reduce((prev, item) => {
        const [key, value] = item.split(keyValueSeparator);
        
	prev[key] = key && !value ? true : value;        

        return prev;  
    }, {});
}
``` 
or smth like that.

Reply:
Maybe we can use [yargs](https://github.com/yargs/yargs) instead of `commander` (its syntax for simple options [is flexible]()), and just use the [Object notation](https://github.com/yargs/yargs/blob/master/docs/tricks.md#objects) for complex arguments?

Reply:
Wow, we can use [querystring.parse](https://nodejs.org/api/querystring.html#querystring_querystring_parse_str_sep_eq_options) with the customized `sep` to parse lists of key-value pairs, brilliant!

Reply:
Nah, it doesn't play well will with treating values as simple boolean flags.